### DISCUSSION TITLE
Position of `<scordatura>` element

### DISCUSSION THREAD

**Discussion by lemzwerg** [w3c-cg/musicxml#513] [Q&A]
By default, a scordatura is to be displayed at the very beginning of music, before anything else. I now wonder whether the `<scordatura>` element is expected to be positioned before the `<attributes>` element within a measure or vice versa – should an application reorder the elements (i.e., scordatura, clef, key, and time signature) or follow the order given in the MusicXML file?

**Comment by mscuthbert** [comment]
I'll continue the discussion by saying I have never thought about the placement of this element so I'll follow the discussion before chiming in but I do ask everyone to consider scordatura with multiple staves or voices and backup elements in your discussion. I don't want to make it impossible to write harp etc. scordatura even while making more normal cases clearer and easier.
